### ISSUE TITLE
Bug in scientific e notation numbers with negative exponents

### ISSUE THREAD

**Issue by scorpion9979** [shrpne/from-exponential#7]
Let `a` be equal to `fromExponential("1.2345e-10")`.
Let `b` be equal to `fromExponential("12.345e-10")`.

While in fact b = a * 10, there currently exists a bug that results in `a === b` being true.

**Comment by PaulRBerg** [comment]
Can confirm that this bug exists. The following code:

```typescript
import fromExponential from "from-exponential";

const a: string = fromExponential("1.2345e-10");
const b: string = fromExponential("12.345e-10");

console.log({ a, b });
```

Logs the following:

```js
{ a: '0.00000000012345', b: '0.00000000012345' }
```

**Comment by PaulRBerg** [comment]
You can add this test to your "positive exponential-like strings" testing context:

```js
expect(fromExponential('12.123e-5')).toEqual('0.00012123');
```

The tests will fail with the following error:

```js
expect(received).toEqual(expected) // deep equality

    Expected: "0.00012123"
    Received: "0.000012123"
```

**Comment by PaulRBerg** [comment]
Interestingly, the bug is applicable only when the inputs are stringified. When they are passed as normal JavaScript numbers, the outputs are correct - probably because the [String constructor](https://github.com/shrpne/from-exponential/blob/master/src/helpers.js#L7) performs the conversion from exponential form automatically.

**Comment by PaulRBerg** [comment]
Just made a PR for this: #8.

**Comment by shrpne** [comment]
Fix released under [v1.1.1](https://github.com/shrpne/from-exponential/releases/tag/v1.1.1)
